install-dev/upgrade/sql/1.7.8.1.sql

```diff
@@ -0,0 +1,7 @@
+SET SESSION sql_mode='';
+SET NAMES 'utf8mb4';
+
+INSERT INTO `PREFIX_product_supplier` (id_product, id_supplier, product_supplier_reference, product_supplier_price_te, id_currency)
```

Comment:
does it require to be updated? It's related to fixtures data.
ALso, the update file must be defined in the autoupgrade module

Reply:
You're right, it must be defined in the autoupgrade module.
I still think it must be updated though, because it's easy to do and if for whatever reason there is still  fixture products associated with a supplier, the issue will occur.